### PR TITLE
load PFR

### PR DESCRIPTION
**Pull request by guga31bb**
Not ready. Still need documentation and snap counts

### CODE REVIEW

**Comment by guga31bb** [comment]
Why is this failing now...

**Comment by guga31bb** [comment]
- [ ] Wait for Lee to add .rds for draft picks
- [ ] Data dictionaries
- [ ] Tests?

File: R/load_nflverse.R [hunk 1]
```diff
@@ -303,6 +303,92 @@ load_injuries <- function(seasons = most_recent_season(),
...
+#'
+#' @examples
+#' \donttest{
+#'   load_snap_counts()
+#' }
+#'
+#' @return A tibble of season-level player statistics provided by Pro Football Reference.
+#'
+#' @export
+load_snap_counts <- function(seasons = TRUE){
```
Comment (mrcaseb) [inline R/load_nflverse.R:353]: I wonder if we should default to `seasons = most_recent_season()` in this case. Typical use case should be to download only the current season imo.

Comment (guga31bb) [reply R/load_nflverse.R]: Agreed & done

**Comment by guga31bb** [comment]
OKAY this might be finally ready?

File: R/load_nflverse.R [hunk 2]
```diff
@@ -350,9 +350,8 @@ load_pfr_passing <- function(seasons = TRUE){
 #' @return A tibble of season-level player statistics provided by Pro Football Reference.
 #'
 #' @export
-load_snap_counts <- function(seasons = TRUE){
+load_snap_counts <- function(seasons = most_recent_season()){
 
-  if(isTRUE(seasons)) seasons <- 2013:most_recent_season()
```
Comment (mrcaseb) [inline R/load_nflverse.R]: Did you intentionally remove that line? I’d like to keep it to be able to load all seasons with the argument `TRUE`

Comment (guga31bb) [reply R/load_nflverse.R]: Oops that was an accident when fixing the function defaults. Fixed!

**Comment by mrcaseb** [comment]
Sorry @guga31bb one last request:
Could you add lines to your tests for passing and snap counts where you let it error with a character input like [here](https://github.com/nflverse/nflreadr/blob/ad5ff5acd39f5b8a9c2923d4005c3161af690b84/tests/testthat/test-nflverse.R#L46)?

**Comment by guga31bb** [comment]
> Sorry @guga31bb one last request:
> Could you add lines to your tests for passing and snap counts where you let it error with a character input like [here](https://github.com/nflverse/nflreadr/blob/ad5ff5acd39f5b8a9c2923d4005c3161af690b84/tests/testthat/test-nflverse.R#L46)?

Done!

**Comment by mrcaseb** [comment]
Looks good to me now! Any objections @tanho63?

**Comment by tanho63** [comment]
Yep LGTM

**Pull request merged**